docs/Users_Guide/getting_started.rst

```diff
@@ -0,0 +1,356 @@
+      Wavelet-Stat :raw-html:`<br />`
+      MODE :raw-html:`<br />`
+      MTD :raw-html:`<br />`
+      Series-Analysis :raw-html:`<br />`
+      Grid-Diag :raw-html:`<br />`
+      TC-Gen :raw-html:`<br />`
+      TC-RMW :raw-html:`<br />`
+    - Point-Stat (run with the Analyses :raw-html:`<br />`
+      as the “forecast” and Point Forecast :raw-html:`<br />`
+      as the “observation”)
+  * - Point Observations
+    - Point-Stat :raw-html:`<br />`
+      Ensemble-Stat
+    - Stat-Analysis (run by passing in MPR records) :raw-html:`<br />`
+      TC-Pairs :raw-html:`<br />`
+      TC-Gen :raw-html:`<br />`
+      TC-Stat
+
+      
+* What attributes of the forecast should be evaluated?
+  
+  * This refers to not only defining the forecast fields to be evaluated
+    but also the forecast characteristics such as bias, reliability,
+    resolution, and prediction of events.  It also means understanding
+    the nature of the forecast and observations. 
+
+Examples of the nature of fields to be evaluated
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Continuous fields – the values change at the decimal level.
+
+* Categorical fields – the values change incrementally most
+  likely as integers or categories.  Continuous fields can also be
+  turned into categorical fields via applying thresholds.
+  
+* Probability fields – the values represent the probability or
+  likelihood of an event occurring, usually represented by thresholds.
+   
+* Ensemble fields – are made up of multiple predictions either from
+  the same modeling system or multiple systems.
+
+Definitions of statistics categories associated with each type of field:
+ 
+* Continuous statistics - measures how the values of the forecasts
+  differ from the values of the observations.
+   
+  * METplus line types: SL1L2, SAL1L2, VL1L2, VAL1L2, CNT, VCNT.
+     
+  * METplus tools:
+      
+* Categorical statistics - measures how well the forecast captures events.
+   
+  * METplus line types: FHO, CTC, CTS, MCTC, MCTS, ECLV, TC stats,
+    ExtraTC stats, TC-Gen stats.
+    
+* Probability statistics - measures attributes such as reliability,
+  resolution, sharpness, and uncertainty.
+
+  * METplus line types: PCT, PSTD, PJC, PRC.
+     
+* Ensemble statistics - measures attributes as the relationship between
+  rank of observation and members, spread of ensemble member solutions
+  and continuous measures of skill.
+
+Additional verification and diagnostic approaches that can be helpful
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Geographical methods demonstrate where the error occurs geographically.
+
+  * METplus methods: Series-Analysis tool.
+    
+  * METplus line types: Most Grid-Stat and Point-Stat line types.
+    
+* Object Based measures the location error of the forecast and how the
+  total error break down into variety of descriptive attributes.
+   
+  * METplus methods: MODE, MTD, MvMODE, Grid-Stat Distance Maps.
```

Comment:
"MvMODE"? Is that a typo?

Reply:
@TaraJensen could you review MvMODE?  Is this a typo?

Reply:
Thanks @j-opatz!  Given that information, I am thinking maybe we could say Multivariate MODE (MvMODE). Thoughts? @j-opatz @TaraJensen @lisagoodrich